network/ddclient-configuration.sh

```diff
@@ -43,6 +44,14 @@ case "$choice" in
         SERVER="update.dedyn.io"
         USE_SSL="yes"
     ;;
+    "Duck DNS")
+        PROVIDER="Duck DNS"
+        INSTRUCTIONS="get a DDNS account with password"
+        GUIDE="https://www.duckdns.org/faqs.jsp"
```

Comment:
I haven't found a better guide where it is explained only how to retreivew a duckdns account and password inkl domain, etc.